americangut/power_plots.py

```diff
@@ -0,0 +1,581 @@
+        row = ['\t<tr>',
+               '\t\t<td style="border-top:hidden;',
+               '\t\t           border-bottom:hidden;',
+               '\t\t           border-left: hidden;',
+               '\t\t           border-bottom: hidden;',
+               '\t\t           padding:10px;',
+               '\t\t           text-align:left',
+               '\t\t          ">',
+               '\t\t\t%s' % cat.replace('_', ' ').title(),
+               '\t\t</td>',
+               '\t\t<td style="border-top:hidden;',
+               '\t\t           border-bottom:hidden;',
+               '\t\t           border-left: hidden;',
+               '\t\t           border-bottom: hidden;',
+               '\t\t           text-align:right',
+               '\t\t          ">',
+               '\t\t\t%i' % a_eff_fit,
+               '\t\t</td>',
+               '\t\t<td style="border-top:hidden;',
+               '\t\t           border-bottom:hidden;',
+               '\t\t           border-left: hidden;',
+               '\t\t           border-bottom: hidden;',
+               '\t\t           text-align:center',
+               '\t\t          ">',
+               '\t\t\t&plusmn;',
+               '\t\t</td>',
+               '\t\t<td style="border-top:hidden;',
+               '\t\t           border-bottom:hidden;',
+               '\t\t           border-left: hidden;',
+               '\t\t           border-bottom: hidden;',
+               '\t\t           text-align:right',
+               '\t\t          ">',
+               '\t\t\t%i' % a_eff_rnd,
+               '\t\t</td>',
+               '\t\t<td style="border-top:hidden;',
+               '\t\t           border-bottom:hidden;',
+               '\t\t           border-left: hidden;',
+               '\t\t           border-bottom: hidden;',
+               '\t\t           padding:20px">',
+               '\t\t</td>',
+               '\t\t<td style="border-top:hidden;',
+               '\t\t           border-bottom:hidden;',
+               '\t\t           border-left: hidden;',
+               '\t\t           border-bottom: hidden;',
+               '\t\t           text-align:right',
+               '\t\t          ">',
+               '\t\t\t%i' % b_eff_fit,
+               '\t\t</td>',
+               '\t\t<td style="border-top:hidden;',
+               '\t\t           border-bottom:hidden;',
+               '\t\t           border-left: hidden;',
+               '\t\t           border-bottom: hidden;',
+               '\t\t           text-align:center',
+               '\t\t          ">',
+               '\t\t\t&plusmn;',
+               '\t\t</td>',
+               '\t\t<td style="border-top:hidden;',
+               '\t\t           border-bottom:hidden;',
+               '\t\t           border-left: hidden;',
+               '\t\t           border-bottom: hidden;',
+               '\t\t           text-align:right',
+               '\t\t          ">',
+               '\t\t\t%i' % b_eff_rnd,
```

Comment:
just fyi, the vast majority, if not all, of the tab characters aren't doing anything...